mods/common/hotkeys/game.yaml

```diff
@@ -18,6 +18,10 @@ SelectUnitsByType: W
 	Description: Select units by type
 	Types: World
 
+RemoveFromControlGroups: BACKSPACE Ctrl
+	Description: Remove from control groups
```

Comment:
Please make it "Remove from control group", otherwise the text overflows in the dialog. Also please leave the default hotkey empty, as a compromise. :+1: after that.

Reply:
Yeah, i'm happy to let my objections slide if this becomes an option feature that players enable themselves.

Reply:
Changes applied.